docs/graphql-api/subscriptions/index.mdx

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_label: Subscriptions
+sidebar_position: 0
+description:
+  "Comprehensive guide to working with GraphQL Subscriptions and leveraging the real-time capabilities of Hasura's
+  GraphQL API. Explore functionalities like live updates, filtering, and multiple concurrent subscriptions. Learn more
+  about managing subscriptions effectively, and the use of variables, fragments, and directives."
+keywords:
+  - graphql subscriptions
+  - hasura graphql api
+  - hasura subscriptions
+  - real-time api
+  - live updates
+toc_max_heading_level: 4
+---
+
+# GraphQL Subscriptions
+
+GraphQL subscriptions allow you to push real-time updates from your database to clients, making them ideal for building
+reactive applications without constant polling.
+
+Subscriptions are supported for the following queries:
+
+- [Select unique](/supergraph-modeling/models.mdx#model-selectuniquegraphqldefinition)
+- [Select many](/supergraph-modeling/models.mdx#model-selectmanygraphqldefinition)
+- [Aggregate](/supergraph-modeling/models.mdx#model-modelaggregategraphqldefinition)
+
+Example:
+
+```graphql
+subscription UserNotificationSubscription {
+  notifications(where: { user_id: { _eq: 123 } }) {
+    id
+    created_at
+    message
+  }
+}
+```
+
+## Enabling Subscriptions
+
+You can enable subscriptions on your GraphQL API in the following ways:
+
+### Fresh Hasura DDN Project
+
+If you're starting fresh, set up a Hasura DDN project using the [quickstart](/getting-started/quickstart.mdx) guide.
+Once your models are added to the supergraph using the [`ddn model add`](/cli/commands/ddn_model_add.mdx) command,
+subscription capabilities are automatically generated for them. For futher customization, you can manually
+[update the metadata](#manual-metadata-configuration).
```

Comment:
Instead of `manually update the metadata`, I feel more concise language would be `edit the metadata`